components/display/include/display.hpp

```diff
@@ -285,20 +353,39 @@ template <typename Pixel> class Display : public BaseComponent {
    *        display driver. Start the task to run the high-priority lvgl
    *        task.
    * @param flush_callback Callback used to flush color data to the display.
-   * @param rotation_fn function to call in the event handler on rotation change.
+   * @param rotation_callback function to call in the event handler on rotation change.
    * @param rotation Default / initial rotation of the display.
    * @param task_config Configuration for the task that runs the lvgl tick
+   * @param backlight_pin GPIO pin for the backlight. -1 if no backlight.
+   * @param backlight_on_value Value to write to the backlight pin to turn the backlight on.
    */
-  void init(flush_fn flush_callback, rotation_fn rotation_callback, DisplayRotation rotation,
-            const Task::BaseConfig &task_config) {
+  void init(const flush_fn flush_callback, const rotation_fn rotation_callback,
+            DisplayRotation rotation, const Task::BaseConfig &task_config,
+            const gpio_num_t backlight_pin = GPIO_NUM_NC, const bool backlight_on_value = true) {
+    if (backlight_pin != -1) {
```

Comment:
I know they're the same, but please make this a check against `GPIO_NUM_NC` instead of `-1`